config/config.go

```diff
@@ -24,6 +24,7 @@ type Config struct {
 	OTExporterOTLPEndpoint     string        `envconfig:"OTEL_EXPORTER_OTLP_ENDPOINT"`
 	OTServiceName              string        `envconfig:"OTEL_SERVICE_NAME"`
 	OTBatchTimeout             time.Duration `envconfig:"OTEL_BATCH_TIMEOUT"`
+	OtelEnabled                bool          `envconfig:"OTEL_ENABLED"`
```

Comment:
Just need to add this to [the README](https://github.com/ONSdigital/dp-population-types-api/blob/feature/feature-flag-otel/README.md#configuration)